apps/prairielearn/src/sprocs/assessment_groups_add_member.sql

```diff
@@ -12,6 +12,7 @@ BEGIN
     -- ##################################################################
     -- verify the updating group belongs to the selected assessment
     -- then lock the group row
+    -- TODO: This sproc and associated UI doesn't actually respect maximum size constraints.
```

Comment:
🙃 I didn't necessarily want to address this here. This constraint checking isn't as important, as only instructors are currently able to make group membership exceed the max size.